src/app/components/elements/EventBookingForm.tsx

```diff
@@ -14,6 +14,112 @@ interface EventBookingFormProps {
   updateAdditionalInformation: (update: AdditionalInformation) => void;
 }
 
+const SchoolYearGroup = ({
+  event,
+  additionalInformation,
+  updateAdditionalInformation,
+}: {
+  event: AugmentedEvent;
+  additionalInformation: AdditionalInformation;
+  updateAdditionalInformation: (update: AdditionalInformation) => void;
+}) => {
+  return (
+    <>
+      <RS.Label htmlFor="year-group" className="form-required">
+        School year group
+      </RS.Label>
+      <RS.Input
+        type="select"
+        id="year-group"
+        name="year-group"
+        value={additionalInformation.yearGroup || ""}
+        onChange={(event) => updateAdditionalInformation({ yearGroup: event.target.value })}
+      >
+        <option value="" />
+        {event.isAStudentEvent && (
+          <>
+            <option value="9">Year 9</option>
+            <option value="10">Year 10</option>
+            <option value="11">Year 11</option>
+            <option value="12">Year 12</option>
+            <option value="13">Year 13</option>
+          </>
+        )}
+        {!event.isStudentOnly && (
+          <>
+            <option value="TEACHER">N/A - Teacher</option>
+            <option value="OTHER">N/A - Other</option>
+          </>
+        )}
+      </RS.Input>
+      {event.isStudentOnly && (
+        <div className="text-muted" data-testid="student-only">
+          {studentOnlyEventMessage(event.id)}
+        </div>
+      )}
+      {(event.isATeacherEvent || additionalInformation.yearGroup == "TEACHER") && (
+        <div className="mt-2 text-right">
+          <a href="/pages/teacher_accounts" target="_blank">
+            Click to upgrade to a teacher account
+          </a>{" "}
+          for free!
+        </div>
+      )}
+    </>
+  );
+};
+
+export const AccessibilityAndMedicalRequirements = ({
+  additionalInformation,
+  updateAdditionalInformation,
+}: {
+  additionalInformation: AdditionalInformation;
+  updateAdditionalInformation: (update: AdditionalInformation) => void;
+}) => {
+  const data = [
+    {
+      label: "Accessibility requirements",
+      id: "access",
+      popover:
+        "For example, please let us know if you need wheelchair access, hearing loop or if we can help with any special adjustments.",
+      type: "accessibilityRequirements",
+    },
+    {
+      label: "Dietary requirements or relevant medical conditions",
+      id: "medical",
+      popover:
+        "For example, please let us know if you need wheelchair access, hearing loop or if we can help with any special adjustments.",
+      type: "medicalRequirements",
+    },
+  ];
+
+  return (
+    <div>
+      {data.map((each) => {
```

Comment:
Small one, but `each` is not the best name. The same goes for `data`. They seem to be requirements, so perhaps the right names would be `requirements` (or `requirementList`) instead of `data` and `requirement` instead of `each`. It's just a suggestion, feel free to change it to something more accurate/informative than requirement.